src/makeCalculator.js

```diff
@@ -37,7 +37,32 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(num) {
+      this.result += num;
```

Comment:
It would be good to add data validation: to check if the input is a number before performing operations.